### PR TITLE
Activate on focus changed callback (iOS)

### PR DESCRIPTION
**Pull request by ju5td0m7m1nd**
add event listener for focusChanged event (iOS)

### CODE REVIEW

**Comment by lodev09** [comment]
Someone merge this please? `onFocusChanged` and `onZoomChanged` is not called in ios. Thanks!

File: Example/ios/Example.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -379,6 +379,9 @@
 						CreatedOnToolsVersion = 6.2;
 						TestTargetID = 13B07F861A680F5B00A75B9A;
 					};
+					13B07F861A680F5B00A75B9A = {
+						DevelopmentTeam = P3ZUK8AHN7;
```
Comment (ptomasroos) [inline Example/ios/Example.xcodeproj/project.pbxproj]: This does not belong here

File: Example/ios/Example.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -648,7 +651,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CURRENT_PROJECT_VERSION = 1;
 				DEAD_CODE_STRIPPING = NO;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = P3ZUK8AHN7;
```
Comment (ptomasroos) [inline Example/ios/Example.xcodeproj/project.pbxproj]: Same here.

File: Example/ios/Example.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -672,7 +675,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = P3ZUK8AHN7;
```
Comment (ptomasroos) [inline Example/ios/Example.xcodeproj/project.pbxproj]: And here

**Comment by KGALLET** [comment]
Can this PR be merge ?

**Pull request closed without merging**